docs/Installation/QuickStartWithDockerAndVSCode.md

```diff
@@ -0,0 +1,567 @@
+`command:` below to match your spectre clone's path.*** *Also change `-j4`
+depending on the number of processors and amount of RAM available to
+docker on your system.*
+~~~~
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Configure spectre",
+            "type": "shell",
+            "command": "bash /work/cmake_spectre.sh"
+        },
+        {
+            "label": "Build spectre",
+            "type": "shell",
+            "command": "cd /work/spectre-build-clang; make -j4",
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            }
+        },
+        {
+            "label": "Build spectre test-executables",
+            "type": "shell",
+            "command": "cd /work/spectre-build-clang; make test-executables",
+            "group": "build"
+        },
+        {
+            "label": "Test spectre",
+            "type": "shell",
+            "command": "cd /work/spectre-build-clang;ctest --output-on-failure",
+            "group": {
+                "kind": "test",
+                "isDefault": true
+            }
+        }
+    ]
+}
+~~~~
+
+* `.vscode/launch.json`. This file configures for using Visual Studio Code's
+debugger. Later on, you can change the `args:` option to choose a different
+test to debug, or change the `program` and `args` tags for whatever
+executable you would like to debug. This example would debug the unit test
+`Unit.ApparentHorizons.StrahlkorperGr.RicciScalar`.
+~~~~
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "(gdb) Launch",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "/work/spectre-build-clang/bin/RunTests",
+            "args": ["Unit.ApparentHorizons.StrahlkorperGr.RicciScalar"],
+            "stopAtEntry": true,
+            "cwd": "${workspaceFolder}",
+            "environment": [],
+            "externalConsole": false,
+            "MIMode": "gdb",
+            "setupCommands": [
+                {
+                    "description": "Enable pretty-printing for gdb",
+                    "text": "-enable-pretty-printing",
+                    "ignoreFailures": true
+                }
+            ]
+        }
+    ]
+}
+~~~~
+
+Finally, add the shell script used to run `cmake`. Create a file
+`/work/cmake_spectre.sh` and put the following into it, replacing the last line
+with the path to your spectre clone. One way to do this is to open a terminal
+and run `code /work/spectre-build-clang/cmake_spectre.sh`
+~~~~
+#!/bin/bash
+cd /work/spectre-build-clang
+cmake -D CMAKE_CXX_COMPILER=clang++ -D CMAKE_C_COMPILER=clang \
+-D CMAKE_Fortran_COMPILER=gfortran-8 \
+-D CHARM_ROOT=/work/charm/multicore-linux64-clang \
+/Users/geoffrey/Codes/spectre/spectre-alt
+
+~~~~
+
+## Edit git configuration
```

Comment:
The git config is forwarded from the host to the container, so the user should set it before even git-cloning the repository.